Mathlib/MeasureTheory/Function/LpSpace.lean

```diff
@@ -835,11 +836,18 @@ theorem dist_indicatorConstLp_eq_norm {t : Set α} {ht : MeasurableSet t} {hμt
 @[simp]
 theorem indicatorConstLp_empty :
     indicatorConstLp p MeasurableSet.empty (by simp : μ ∅ ≠ ∞) c = 0 := by
-  rw [Lp.eq_zero_iff_ae_eq_zero]
-  convert indicatorConstLp_coeFn (E := E)
-  simp [Set.indicator_empty', Pi.zero_def]
+  simp only [indicatorConstLp, Set.indicator_empty', Memℒp.toLp_zero]
 #align measure_theory.indicator_const_empty MeasureTheory.indicatorConstLp_empty
 
+theorem indicatorConstLp_injective {s t : Set α} (hs : MeasurableSet s) (hsμ : μ s ≠ ∞)
```

Comment:
Shouldn't this be spellt `indicatorConstLp_inj`? (I think the `injective` version is for when you use `Function.Injective ...`).